components/UI/Map/LeafletMap.tsx

```diff
@@ -55,6 +55,26 @@ const MapEvents = () => {
   const mapZoomLevelRef = useRef(0);
   const { setCoordinates, setPopUpData } = useMapActions();
 
+  useEffect(() => {
+    const localCoordinatesURL = window.localStorage.getItem("coordinatesURL");
```

Comment:
Bu key'i bir constant mi yapsak, ileride birilerinin typo yapmasi olasi. 

```suggestion
    const localCoordinatesURL = window.localStorage.getItem("coordinatesURL");
```

Reply:
fixledim simdi, tesekkurler